x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/search_after_bulk_create.ts

```diff
@@ -49,23 +49,33 @@ export const searchAfterAndBulkCreate = async ({
 }: SearchAfterAndBulkCreateParams): Promise<SearchAfterAndBulkCreateReturnType> => {
   return withSecuritySpan('searchAfterAndBulkCreate', async () => {
     let toReturn = createSearchAfterReturnType();
+    let searchingIteration = 0;
 
     // sortId tells us where to start our next consecutive search_after query
     let sortIds: estypes.SortResults | undefined;
     let hasSortId = true; // default to true so we execute the search on initial run
 
     if (tuple == null || tuple.to == null || tuple.from == null) {
-      ruleExecutionLogger.error(`[-] malformed date tuple`);
+      ruleExecutionLogger.error(
+        `missing run options fields: ${!tuple.to ? '"tuple.to"' : ''}, ${
+          !tuple.from ? '"tuple.from"' : ''
+        }`
+      );
       return createSearchAfterReturnType({
         success: false,
         errors: ['malformed date tuple'],
       });
     }
 
     while (toReturn.createdSignalsCount <= tuple.maxSignals) {
+      const cycleNum = `cycle ${searchingIteration++}`;
```

Comment:
We might want to add documentation somewhere for what a `cycle` is in this instance if we're exposing these logs a lot more. At least for internal use. Especially since we could have multiple `cycle 1, 2, etc...` per tuple in the rule run